classes/ActionScheduler_ListTable.php

```diff
@@ -0,0 +1,378 @@
+		return '<code>' . json_encode( $row['args'] ) . '</code>';
+	}
+
+	/**
+	 * Prints the logs entries inline. We do so to avoid loading Javascript and other hacks to show it in a modal.
+	 *
+	 * @param array $row Action array.
+	 */
+	public function column_log( array $row ) {
+		echo '<ol>';
+		$timezone = new DateTimezone( 'UTC' );
+		foreach ( $row['log'] as $log ) {
+			$date = $log->get_date();
+			$date->setTimezone( $timezone );
+			echo '<li><strong>' . esc_html( $date->format( 'Y-m-d H:i:s e' ) ) . '</strong><br/>' . esc_html( $log->get_message() ) . '</li>';
+		}
+		echo '</ol>';
+	}
+
+	/**
+	 * Renders admin notifications
+	 *
+	 * Notifications:
+	 *  1. When the maximum number of tasks are being executed simultaneously
+	 *  2. Notifications when a task us manually executed
+	 */
+	public function maybe_render_admin_notices() {
+		if ( self::$did_notification ) {
+			return;
+		}
+
+		self::$did_notification = true;
+
+		if ( $this->store->get_claim_count() >= apply_filters( 'action_scheduler_queue_runner_concurrent_batches', 5 ) ) : ?>
+<div id="message" class="updated">
+	<p><?php printf( __( 'Maximum simulatenous batches already in progress (%s queues). No actions will be processed until the current batches are complete.', 'action-scheduler' ), $this->store->get_claim_count() ); ?></p>
+</div>
+		<?php endif;
+		$notification = get_transient( 'actionscheduler_admin_executed' );
+		if ( is_array( $notification ) ) {
+			delete_transient( 'actionscheduler_admin_executed' );
+
+			$action = $this->store->fetch_action( $notification['action_id'] );
+			$action_hook_html = '<strong>' . $action->get_hook() . '</strong>';
+			if ( 1 == $notification['success'] ): ?>
+				<div id="message" class="updated">
+					<p><?php printf( __( 'Successfully executed the action: %s', 'action-scheduler' ), $action_hook_html ); ?></p>
+				</div>
+			<?php else : ?>
+			<div id="message" class="error">
+				<p><?php printf( __( 'Could not execute the action: %s', 'action-scheduler' ), $action_hook_html ); ?></p>
+			</div>
+			<?php endif;
+		}
+	}
+
+
+	/**
+	 * Prints the scheduled date in a human friendly format.
+	 *
+	 * @param array $row The array representation of the current row of the table
+	 */
+	public function column_scheduled( $row ) {
+		if ( $row['scheduled']->next() ) {
+			$next_timestamp = $row['scheduled']->next()->format( 'U' );
+			echo $row['scheduled']->next()->format( 'Y-m-d H:i:s' );
+			if ( gmdate( 'U' ) > $next_timestamp ) {
+				printf( __( ' (%s ago)', 'action-scheduler' ), human_time_diff( gmdate( 'U' ), $next_timestamp ) );
+			} else {
+				echo ' (' . human_time_diff( gmdate( 'U' ), $next_timestamp ) . ')';
+			}
+		}
+	}
+
+	/**
+	 * Bulk delete
+	 *
+	 * Deletes actions based on their ID. This is the handler for the bulk delete. It assumes the data
+	 * properly validated by the callee and it will delete the actions without any extra validation.
+	 *
+	 * @param array $ids
+	 *
+	 * @return void
```

Comment:
It sounds like it's actually the recommended practice, not just a personal preference: https://stackoverflow.com/a/2061559

As we have a lot of instances of this, I've created #107 to address them. I'd prefer this instance be removed as part of that rather than rebasing its removal into Cesar's commit.

Reply:
Sounds good to me 👍